app/models/comment.rb

```diff
@@ -105,6 +106,6 @@ def user_submitted?
     end
 
     def authored_by_seller?
-      commentable.respond_to?(:seller_id) && commentable.seller_id == author_id
```

Comment:
No need to check for `respond_to?(:seller_id)` any longer since we now check it on line 97.